src/Stratis.Bitcoin/Configuration/NodeSettings.cs

```diff
@@ -361,10 +361,12 @@ private string CreateDefaultDataDirectories(string appName, Network network)
         /// <summary>
         /// Displays command-line help.
         /// </summary>
-        /// <param name="network">The network to extract values from.</param>
-        public static void PrintHelp(Network network)
+        /// <param name="featureTypes">The list of features to display help for.</param>
+        /// <param name="serviceProvider">The service provider that can resolve feature types.</param>
+        public static void PrintHelp(List<Type> featureTypes, IServiceProvider serviceProvider)
         {
-            Guard.NotNull(network, nameof(network));
+            var nodeSettings = (NodeSettings)serviceProvider.GetService(typeof(NodeSettings));
```

Comment:
Network can be passed in to the metohd (no need to use DI)

Reply:
Changed.